cxx-qt-gen/src/gen_rs.rs

```diff
@@ -478,11 +359,8 @@ pub fn generate_qobject_cxx(
     // Build the import path for the C++ header
     let import_path = format!("cxx-qt-gen/include/{}.cxxqt.h", ident_snake);
 
-    // TODO: ideally we only want to add the "type QString = cxx_qt_lib::QString;"
-    // if we actually generate some code that uses QString.
-
     // Build the namespace string, rust::module
```

Comment:
comment no longer needed? I think the code is self-explanatory now

Reply:
Removed.